backend/app.js

```diff
@@ -38,6 +39,9 @@ app.get("/api/v2/lumens", lumensV2.handler);
 app.get("/api/v2/lumens/total-supply", lumensV2.totalSupplyHandler);
 app.get("/api/v2/lumens/circulating-supply", lumensV2.circulatingSupplyHandler);
```

Comment:
❓ Why don't we have the circulating supply and `/api/v2/lumens` on the `v3`? If we keep functionality only available in old versions we may struggle to get people off old versions.

Reply:
agreed this was needed to https://github.com/stellar/dashboard/pull/182/commits/85d77b3767787db56bf755c998ddf44cb630f619

Reply:
What about `/api/v2/lumens`?

Reply:
☝🏻

Reply:
oh hmm it would be exactly the same. if we make that one v3 does it imply something changed?

Reply:
We should have every endpoint in the new api version, otherwise we are saying anything not migrated is deprecated.

Reply:
I'm not sure if these are all or nothing changes. Eg. just updating the endpoints that changed makes sense in my head. But I'll defer if you think that one should be upped a version too

Reply:
I assume `/api/ledgers/public` was deprecated when we released v2. But if it wasn't, we should include it too. @accordeiro Do you know?

Reply:
> I assume /api/ledgers/public was deprecated when we released v2. But if it wasn't, we should include it too. @accordeiro Do you know?

note: I'll be working on updating the backend of this repo so I could do this cleanup/add to v3 as part of that